EIPS/eip-wrapped_token.md

```diff
@@ -0,0 +1,124 @@
+---
+eip: <to be assigned>
+title: Wrapped token inspection
+author: David Mihal (@dmihal)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category (*only required for Standard Track): ERC
+created: 2020-08-05
+---
+
+## Simple Summary
+Defines a standard interface for viewing the addresses and balances of wrapped tokens.
+
+## Abstract
+Many projects have emerged that create new assets by wrapping existing tokens. However, it can be difficult for applications to easily query which tokens are wrapped, and how much of each token is associated with the holder of the wrapper token.
+
+This EIP introduces a simple interface for exposing the addresses of all wrapped tokens, as well as querying wrapped token balances.
```

Comment:
This content should all be moved to the Motivation section.  The Abstract should be a terse human readable version of the specification.